packages/react-router-dom/jest.config.js

```diff
@@ -20,4 +21,5 @@ module.exports = {
     "^react-router$": "<rootDir>/../react-router/index.ts",
     "^react-router-dom$": "<rootDir>/index.tsx",
   },
+  reporters: ["default"],
```

Comment:
explicitly setting the reporter makes it so it doesn't get set to `github-actions` which will do annotations on the "files changed" tab

https://jestjs.io/docs/configuration#reporters-arraymodulename--modulename-options